model/github_runner.rb

```diff
@@ -36,10 +36,13 @@ def display_state
   end
 
   def log_duration(message, duration)
-    values = {ubid: ubid, label: label, repository_name: repository_name, duration: duration}
+    values = {ubid: ubid, label: label, repository_name: repository_name, duration: duration, conclusion: workflow_job&.dig("conclusion")}
```

Comment:
Do we need conclusion info each point we log duration? We might have additional log_conclusion function and call it from the billing record. Also, logging conclusion from log_duration function is a bit unintuitive.

Reply:
Actually, the duration is the execution time for `runner_completed` log. So it's not just conclusion log.

Reply:
`workflow_job&.dig("conclusion")` is `nil` if the job not completed yet

Reply:
I can add a condition check if `{ conclusion: nil }` feels wrong such as

```ruby
if (conclusion = workflow_job&.dig("conclusion"))
  values[:conclusion] = conclusion
end
```

Reply:
@velioglu Any comment?